src/java.base/share/classes/module-info.java

```diff
@@ -280,7 +283,8 @@
         jdk.sctp;
     exports sun.net.www to
         java.net.http,
-        jdk.jartool;
+        jdk.jartool,
+        openjceplus;
```

Comment:
This one is somewhat surprising; we really ought to limit the need for these qualified exports. Can you please double-check that _all_ the additions here are absolutely necessary?

Reply:
I tried to remove this `sun.net.www `exports to openjceplus, but got below error. So, the `sun.net.www` package is needed when building openjceplus. 

```
/home/tao/mkJDK/openj9-openjdk-jdk17/OpenJCEPlus/src/main/openjceplus/share/classes/com/ibm/security/util/PropertyExpander.java:99: error: package sun.net.www is not visible
						val = sun.net.www.ParseUtil.encodePath(val);
						             ^
  (package sun.net.www is declared in module java.base, which does not export it to module openjceplus)
/home/tao/mkJDK/openj9-openjdk-jdk17/OpenJCEPlus/src/main/openjceplus/share/classes/com/ibm/security/util/PropertyExpander.java:102: error: package sun.net.www is not visible
					val = sun.net.www.ParseUtil.encodePath(val);
					             ^
  (package sun.net.www is declared in module java.base, which does not export it to module openjceplus)
```

But I re-checked all the exports in module-info, and removed the unnecessary exports for package `com.sun.crypto.provider`, `sun.security.jca`, `sun.security.provider`, `sun.security.x509` and `openj9.internal.security`.